internal/guest/runtime/runc/runc.go

```diff
@@ -251,30 +252,31 @@ func (c *container) GetState() (*runtime.ContainerState, error) {
 // It should be noted that containers that have stopped but have not been
 // deleted are still considered to exist.
 func (c *container) Exists() (bool, error) {
-	states, err := c.r.ListContainerStates()
+	// use global path because container may not exist
+	logPath := c.r.getGlobalLogPath()
+	args := []string{"state", c.id}
+	cmd := createRuncCommand(logPath, args...)
+	out, err := cmd.CombinedOutput()
```

Comment:
Could we just call `GetState` here?

Reply:
`GetState` uses the log file associated with the container in its container directory(`c.r.getLogPath(c.id)`), so if that container is deleted, `GetState` will error because it cannot find that file

Reply:
Hmm I see. We could consider refactoring so these two functions can call the same helper but not necessary I think.